.travis.yml

```diff
@@ -6,7 +6,8 @@ python:
   - "3.7"
   - "3.8"
   - "3.9"
-  - "3.10-dev"
+  - "3.10"
+  - "3.11-dev"
```

Comment:
Hey Paul!

Seems like Python `3.10` and `3.11-dev` are only available in Ubuntu 18.04 and 20.04. 
Could you try adding this to the root of the file:
```
dist: bionic
```
sources: 
- https://docs.travis-ci.com/user/languages/python/#python-versions
- https://docs.travis-ci.com/user/reference/bionic/